assets/robotest/current/status.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: echoserver-status
+  namespace: default
+spec:
+  template:
+    metadata:
+      name: echoserver-status
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - name: echoserver-status
+          image: quay.io/gravitational/debian-tall:buster
+          command:
+            - sh
```

Comment:
Nit: I would use absolute path (`/bin/sh`) to avoid issues if the container doesn't have proper PATH set for some reason. Same for `wget`.

Reply:
Fixed in 2a82ca9.